lxd/instance/drivers/driver_qemu.go

```diff
@@ -5547,6 +5547,11 @@ func (d *qemu) Info() instance.Info {
 		Error: fmt.Errorf("Unknown error"),
 	}
 
+	if !shared.PathExists("/dev/kvm") {
+		data.Error = fmt.Errorf("kvm kernel module not loaded")
```

Comment:
Maybe `KVM support is missing`? As the module could be loaded with the platform not supporting it.